<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputNOForm</class>
 <widget class="QWidget" name="OutputNOForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="TitleBackground" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>5</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Outputting Nitric Oxide</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="DiagLeft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DiagRight">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="diagLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diagnostics</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NOOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>201</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NO Output:</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ppb</string>
   </property>
  </widget>
  <widget class="QLabel" name="flowIconLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tempIconLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lightIconLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>93</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="WaitLabel">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>-5</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
